includes/admin/legacy/class-wc-amazon-payments-advanced-order-admin-legacy.php

```diff
@@ -42,8 +42,9 @@ public function do_order_action( $order, $id, $action, $version ) {
 				$this->clear_stored_states( $order_id );
 				break;
 			case 'authorize':
-				delete_post_meta( $order_id, 'amazon_authorization_id' );
-				delete_post_meta( $order_id, 'amazon_capture_id' );
+				$order->delete_meta_data( 'amazon_authorization_id' );
```

Comment:
Will this be compatible with older supported WooCommerce versions?

Reply:
Yea we require at least WC 4.0 right now and that method was introduced in WC 2.6.0 as stated in woocommerce/includes/abstracts/abstract-wc-data.php WC_Data::delete_meta_data doc comment

Reply:
I see, thanks for explaining!